src/blog/templates/blog/includes/comments.html

```diff
@@ -172,7 +179,17 @@ <h2 class="text-2xl">Comments</h2>
         if (comment.liked_by_user) {
             likeButton.classList.add("text-red-500");
         }
-        likeButton.innerHTML = `<span class="text-xl">😻</span> <span class="like-count">${comment.like_count}</span>`;
+        
+        likeButton.innerHTML = `
+                <div class="text-xl">😻</div>
+                <div class="group">
+                    <div id="comment-like-count" class="like-count">${comment.like_count}</div>
+                    <ul id="comment-like-list-${comment.id}"                               
+                        class="absolute bottom-0 left-4 min-w-24 w-max max-h-24 overflow-auto flex flex-col items-start z-30 px-4 py-2 bg-blue-500 rounded-3xl opacity-0 pointer-events-none hover:opacity-100 group-hover:opacity-100 group-hover:pointer-events-auto group-hover:drop-shadow transition">
+                    </ul>
+                </div>
+        `;
```

Comment:
If you want to make it really easy and don't care about cosmetics, you can also just set the `title` attribute to the list of user's names. So something like:

```js
// pseudo code but you get the gist

// create a single string of like user's names with line breaks
const names = likes.map((like) => `- ${like.display_name} \n);

// set title attribute of like button
likeButton.setAttribute('title', names);
```